plugins/modules/packaging/os/portage.py

```diff
@@ -506,7 +515,12 @@ def main():
     )
 
     module.emerge_path = module.get_bin_path('emerge', required=True)
-    module.equery_path = module.get_bin_path('equery', required=True)
+    try:
+        import portage
```

Comment:
Importing things in the middle of the control flow is an antipattern. A typical idiom used virtually everywhere is having try-except at the top of the module and then having if-else here.